api/graphite.go

```diff
@@ -1027,6 +1027,7 @@ func (s *Server) graphiteTagFindSeries(ctx *middleware.Context, request models.G
 	}
 	seriesNames := make([]string, 0, len(series))
 	for _, serie := range series {
+		// note: for findByTag the "Pattern" is the full metric name
```

Comment:
actually, turns out this is NameWithTags, @replay!

Reply:
Yeah I know, I thought that's what I said